src/kokkos_abstraction.hpp

```diff
@@ -450,6 +468,25 @@ inline void par_for(LoopPatternSimdFor, const std::string &name, DevExecSpace ex
   Kokkos::Profiling::popRegion();
 }
 
+// 1D  outer parallel loop using Kokkos Teams
+template <typename Function>
+inline void par_for_outer(OuterLoopPatternTeams, const std::string &name,
+                          DevExecSpace exec_space, size_t scratch_size_in_bytes,
+                          const int scratch_level, const int kl, const int ku,
```

Comment:
I would recommend using the IndexRange struct for kl and ku, so as to avoid errors associated with the order.

Reply:
Opening another issue as all wrappers would benefit from that #185